library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -613,21 +550,22 @@ void PlatformBridgeFilter::onResumeEncoding() {
         *Buffer::BridgeFragment::createBridgeFragment(*result.pending_data));
     free(result.pending_data);
   } else if (result.pending_data) {
-    Buffer::OwnedImpl inject_data;
-    inject_data.addBufferFragment(
-        *Buffer::BridgeFragment::createBridgeFragment(*result.pending_data));
-    encoder_callbacks_->addEncodedData(inject_data, /* watermark */ false);
-    free(result.pending_data);
-  }
-  if (pending_response_trailers_) {
+    PANIC("invalid filter state: data injection is unsupported at present");
```

Comment:
Should this be an assert instead? Not sure how often we use `PANIC`

Reply:
`PANIC` is effectively equivalent to `RELEASE_ASSERT(false,...)` and it is used, but in this case that's more an artifact of the code I commented out, and it could be re-written as a `RELEASE_ASSERT`. I'll go ahead and do so.